orchagent/neighorch.cpp

```diff
@@ -581,3 +597,129 @@ bool NeighOrch::removeNeighbor(const NeighborEntry &neighborEntry)
 
     return true;
 }
+
+bool NeighOrch::mapToErrorDbFormat(sai_object_type_t& object_type, std::vector<FieldValueTuple> &asicValues,
```

Comment:
Just a suggestion but it might be helpful to break this up based on the type of interface for readability.

Reply:
I would like to leave it as is, as we have only two types of interfaces